### PR TITLE
Update dlp_service_client.py to be able to install it in Custom Prediction

### LINKED ISSUE #11: Unable to install google-cloud-dlp in AI Platform with custom prediction
#### Environment details

  - OS type and version: AI Platform prediction
  - Python version:  Python 3.7
  - `google-cloud-dlp` version: 0.13.0

#### Steps to reproduce

To reproduce:
1. Pass in setup.py (google-cloud-dlp) OR
2. Pass the .tar.gz install file:

```
gcloud beta ai-platform versions create {MODEL1_VERSION} \
    --model {MODEL_NAME} \
    --origin gs://{BUCKET_NAME}/{MODEL_DIR} \
    --runtime-version 1.15 \
    --python-version 3.7 \
    --package-uris gs://{BUCKET_NAME}/{PACKAGES_DIR}/sentiment_classifier_dlp-0.1.tar.gz,gs://{BUCKET_NAME}/google-cloud-dlp-0.13.0.tar.gz \
    --prediction-class=custom_prediction.CustomModelPrediction \
    --service-account {SA_NAME}@{PROJECT}.iam.gserviceaccount.com   
```

I get: `{'error': 'Prediction failed: unknown error.'} `
In logs:
```
ERROR:root:Prediction failed: The 'google-cloud-dlp' distribution was not found and is required by the application
Expand all | Collapse all{
 insertId: "5e8e5528000a21fcbfddb3fb"  
 logName: "projects/news-ml-257304/logs/ml.googleapis.com%2Fprimary.stderr"  
 receiveTimestamp: "2020-04-08T22:50:17.008478460Z"  
 resource: {
  labels: {…}   
  type: "cloudml_model_version"   
 }
 textPayload: "ERROR:root:Prediction failed: The 'google-cloud-dlp' distribution was not found and is required by the application"  
 timestamp: "2020-04-08T22:50:16.664060Z"  
}
```
#### Stack trace

```
The 'google-cloud-dlp' distribution was not found and is required by the application"  
```

#### Workaround:

Seems to be a problem for `pkg_resources.get_distribution` when executed it returns an error: "distribution was not found and is required by the application"

1) To be able to install it:

a) I replaced _GAPIC_LIBRARY_VERSION in this [line](https://github.com/googleapis/python-dlp/blob/master/google/cloud/dlp_v2/gapic/dlp_service_client.py#L44)

```
_GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution("google-cloud-dlp").version
```

with
```
_GAPIC_LIBRARY_VERSION = '0.13.1'
```
* Im using 0.13.1 instead of 0.13.0 just for tracking purposes.

b) Rebuild the library using `python setup.py sdist`. 
c) Uploaded .tar.gz to the GCS bucket and define this value under --package-uris when creating the model version. Working sample: gs://news-ml-experiments/google-cloud-dlp-0.13.1.tar.gz

The way AI Platform Prediction install libraries is via:

```
python -m pip install --target=/tmp/custom_lib --no-cache-dir -b /tmp/pip_builds /tmp/custom_code/google-cloud-dlp-0.13.0.tar.gz
```
I assume `pkg_resources.get_distribution` may not have visibility of this when module is installed via `pip install -t`

https://github.com/pypa/setuptools/blob/53c284b325959a6fd177e9d37d8b5dd5824db2b3/pkg_resources/__init__.py#L355

### PR DESCRIPTION
**Pull request by gogasca**
Address: https://github.com/googleapis/python-dlp/issues/11 in which this library is unable to be installed using Custom Prediction.

- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-dlp/issues/new/choose) before writing your code!  Issue #11 
- [x] Ensure the tests and linter pass

```
nox -s unit-3.7
nox > Running session unit-3.7
nox > Creating virtual environment (virtualenv) using python3.7 in .nox/unit-3-7
nox > pip install mock pytest pytest-cov
nox > pip install -e .
nox > py.test --quiet --cov=google.cloud.dlp --cov=google.cloud --cov=tests.unit --cov-append --cov-config=.coveragerc --cov-report= --cov-fail-under=0 tests/unit
............................................................                         
60 passed, 65 warnings in 1.46s
nox > Session unit-3.7 was successful.
```

```
nox -s system-3.7
nox > Running session system-3.7
nox > Creating virtual environment (virtualenv) using python3.7 in .nox/system-3-7
nox > pip install --pre grpcio
nox > pip install mock pytest
nox > pip install -e test_utils
nox > pip install -e .
nox > py.test --quiet tests/system
.ssssssssssssssssssssssssssssssssssssssssssssssssssssssssss                                                                                                               [100%]
1 passed, 58 skipped in 1.36s
nox > Session system-3.7 was successful.
```

- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #11 🦕

### CODE REVIEW

File: google/cloud/dlp_v2/gapic/dlp_service_client.py [hunk 1]
```diff
@@ -40,8 +40,10 @@
 from google.protobuf import empty_pb2
 from google.protobuf import field_mask_pb2
 
-
-_GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution("google-cloud-dlp").version
+try:
+  _GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution("google-cloud-dlp").version
+except pkg_resources.DistributionNotFound:
+  _GAPIC_LIBRARY_VERSION = "0.14.0"
```
Comment (jochen-at-square) [inline google/cloud/dlp_v2/gapic/dlp_service_client.py:46]: 👍 thanks for the work-around. Might make sense to set the version in dlp.py or somewhere more visible?

Comment (jochen-at-square) [reply google/cloud/dlp_v2/gapic/dlp_service_client.py:46]: Also, if you're going to bump the version, should you match that in setup.py?

**Comment by busunkim96** [comment]
This library is generated from the protos, so this change will be erased the next time we re-run the generator. 

The version is held in `setup.py` because the generator is stateless and doesn't know what version the library is at. We put the version in `setup.py` to avoid duplicating it in multiple places.

I think it'd be preferable to figure out what's going wrong when the package is installed with AI Platform. I'll follow up in issue #11.

**Comment by parthea** [comment]
Closing this PR as there are conflicts. The issue is tracked in #11.

**Pull request closed without merging**